### ISSUE TITLE
Any way to save the full response headers / http error codes?

### ISSUE THREAD

**Issue by luckman212** [louislam/uptime-kuma#4948]
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

**I know most efforts are focused on the V2.0 milestone, so I apologize in advance for this question's timing.**

I'm using Kuma 1.23.13 to monitor 1 site that sits behind a cloudflared proxy. The check is an HTTP check that looks like this:

![screenshot_Im1QJdTp](https://github.com/user-attachments/assets/b99aaedb-90a6-479e-9cc5-d8b485fd2463)

I have been getting frequent 530 errors (which come from Cloudflare) during these checks:

![image](https://github.com/user-attachments/assets/a4fbbfaf-8ab1-4ff6-84b9-5c69119002f8)

I found [this page](https://developers.cloudflare.com/support/troubleshooting/cloudflare-errors/troubleshooting-cloudflare-5xx-errors/#error-530) that describes how to troubleshoot them. It says _"HTTP error 530 is returned with an accompanying 1XXX error displayed. Search for the [specific 1XXX](https://developers.cloudflare.com/support/troubleshooting/cloudflare-errors/troubleshooting-cloudflare-1xxx-errors) error for troubleshooting information."_

**TL;DR—** is there any way to see the full error responses, either in the GUI or by querying the sqlite db directly, so that I can see the 1XXX error that is supposedly being returned?

### 📝 Error Message(s) or Log

Request failed with status code 530

### 🐻 Uptime-Kuma Version

1.23.13

### 💻 Operating System and Arch

Ubuntu 24.04 LTS

### 🌐 Browser

Chrome Version 126.0.6478.183

### 🖥️ Deployment Environment

- Runtime: Docker version 24.0.5
- Database: sqlite/embedded
- Filesystem used to store the database on: ext4
- number of monitors: 64 total

**Comment by CommanderStorm** [comment]
No, there currently are no good debugging features build in.
That would be nice though. Our [contribution guide is here](https://github.com/louislam/uptime-kuma/blob/ba448c765aeab91a2e9bfe6c14a031118648c83e/CONTRIBUTING.md)

**Comment by CommanderStorm** [comment]
I would suggest to switch to postman/insomnia/curl for debugging such cases
